src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.cs

```diff
@@ -17,7 +17,6 @@
 using Uno.Extensions;
 using Uno.Helpers;
 using System.Diagnostics;
-using Microsoft.UI.Xaml.Controls;
```

Comment:
This namespace is needed.

I think we should add this line back and then surround line 12 with `#if !HAS_UNO_WINUI`